src/sonic-yang-models/yang-models/sonic-copp.yang

```diff
@@ -0,0 +1,243 @@
+module sonic-copp {
+
+	yang-version 1.1;
+
+	namespace "http://github.com/Azure/sonic-copp";
+	prefix copp;
+
+	description "CoPP YANG Module for SONiC OS";
+
+	revision 2021-03-31 {
+		description
+			"First Revision";
+	}
+
+	container sonic-copp {
+		container COPP_GROUP {
+			list COPP_GROUP_LIST {
+				key name;
+
+				leaf name {
+					type string;
+					description
+						"Control plane policing group name";
+				}
+
+				leaf queue {
+					type uint8 {
+						range "0..47" {
+							error-message "Invalid value passed for queue";
```

Comment:
Any reason why the range is restricted to 48 queues? Is there any restriction imposed in SAI regarding the maximum number of queues?

Reply:
Removed `range "0..47" {`